postgres/parser/sem/tree/expr.go

```diff
@@ -1390,25 +1390,12 @@ const (
 
 // Format implements the NodeFormatter interface.
 func (node *FuncExpr) Format(ctx *FmtCtx) {
-	var typ string
```

Comment:
It doesn't seem like the changes here should relate to anything, since this is a parser construct, and as far as I'm aware, all of our string output comes from the GMS or `server/` layers. I'm assuming if this were reverted, then the tests would fail? Or I guess, how is this change manifesting in the output below? My assumption could be wrong.

Reply:
If I revert these changes then all these column tests I added would fail because they would include the parentheses and column arguments. Zach pointed me to changing the name for functions for `InputExpression` [here](https://github.com/dolthub/doltgresql/blob/main/server/ast/select.go#L163) and that's used for the alias [here in GMS](https://github.com/dolthub/go-mysql-server/blob/main/sql/planbuilder/project.go#L169)

Reply:
That makes sense!